.vscode/settings.json

```diff
@@ -163,6 +166,8 @@
       "rootPath": "packages/aila"
     }
   ],
+  "jest.runMode": "on-demand",
```

Comment:
I wonder if we need this in user settings somewhere? Without this my mac grinds to a halt because on every save it seems to run the entire test suite!

Reply:
Running the whole test quite automatically doesn't sound good. Half-related to this: I think we should decide on some principles for our jest tests to keep them slim. For example I'd be quite keen to ensure that no external requests are made by code under test

Reply:
Agree! I'll move these settings to a personal file for now